### PR TITLE
Add a comment to the yaml demo-pvc-file-system for volumeMode

### PR DESCRIPTION
**Pull request by lisyukanna**

### CODE REVIEW

File: deploy/kubernetes/examples/demo-pvc-file-system.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 metadata:
   name: demo-pvc-file-system
 spec:
-  volumeMode: Filesystem
+  volumeMode: Filesystem  #Optional. Values Filesystem\Block. The default is Filesystem.
```
Comment (oriyarde) [inline deploy/kubernetes/examples/demo-pvc-file-system.yaml:6]: add 1 space between `#` and `Optional`

**Review by oriyarde — APPROVED** [review]
LGTM

File: deploy/kubernetes/examples/demo-pvc-file-system.yaml (hunk 1 above)
Comment (lisyukanna) [reply deploy/kubernetes/examples/demo-pvc-file-system.yaml]: done

**Comment by ArbelNathan** [comment]
so why we need demo-pvc-raw-block.yaml? and should we change the name of the modified file?

**Comment by oriyarde** [comment]
> so why we need demo-pvc-raw-block.yaml? and should we change the name of the modified file?

it was added with raw block support and it will simplify the example tests.
change to what?

**Comment by lisyukanna** [comment]
> so why we need demo-pvc-raw-block.yaml? and should we change the name of the modified file?

what do you mean?

**Comment by ArbelNathan** [comment]
> 
> 
> > so why we need demo-pvc-raw-block.yaml? and should we change the name of the modified file?
> 
> it was added with raw block support and it will simplify the example tests.
> change to what?

to demo-pvc.yaml.. so why the comment if we have both?

**Comment by ArbelNathan** [comment]
> 
> 
> > so why we need demo-pvc-raw-block.yaml? and should we change the name of the modified file?
> 
> what do you mean?

we have similar pvc but with `volumeMode: Block` so if we have comment that say that `volumeMode` can be `Block `or `FileSystem` so do we still need both files?

**Comment by oriyarde** [comment]
> > > so why we need demo-pvc-raw-block.yaml? and should we change the name of the modified file?
> > 
> > 
> > what do you mean?
> 
> we have similar pvc but with `volumeMode: Block` so if we have comment that say that `volumeMode` can be `Block `or `FileSystem` so do we still need both files?

as I mentioned in CSI-1855, the separate yaml doesn't tell that the field is optional and which value is the default.
and as I mentioned here, it will also simplify the example tests.

File: deploy/kubernetes/examples/demo-pvc-file-system.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 metadata:
   name: demo-pvc-file-system
 spec:
-  volumeMode: Filesystem
+  volumeMode: Filesystem   # Optional. Values Filesystem\Block. The default is Filesystem.
```
Comment (ArbelNathan) [inline deploy/kubernetes/examples/demo-pvc-file-system.yaml:6]: please remove `Values Filesystem\Block.`:
`# Optional. The default is Filesystem.`

Comment (oriyarde) [reply deploy/kubernetes/examples/demo-pvc-file-system.yaml:6]: LGTM as well

Comment (lisyukanna) [reply deploy/kubernetes/examples/demo-pvc-file-system.yaml]: done

**Review by oriyarde — APPROVED** [review]

**Review by ArbelNathan — APPROVED** [review]

**Pull request merged**